vignettes/sir_models.Rmd

```diff
@@ -219,7 +219,7 @@ model <- gen_age$new(data = list(dt = dt,
                      seed = 1L)
 
 # Define how long the model runs for, number of steps
-n_steps <- 300
+n_steps <- 1000
```

Comment:
is it possible to keep this relatively small? It helps keep the CI time sensible as this is run every time the package gets pushed

Reply:
I think this is ok time-wise because it's a small model and only five particles, so the whole chunk takes < 1s on my machine. I believe most of the CI time is compilation (2m 30s on Ubuntu)

Reply:
(reason for length is to show the whole epidemic, keeping comparable with the simple model above. Extra time needed as now population size =~ 60M)